React/Base/RCTConvert.m

```diff
@@ -709,35 +819,50 @@ + (UIColor *)UIColor:(id)json
     CGFloat g = ((argb >> 8) & 0xFF) / 255.0;
     CGFloat b = (argb & 0xFF) / 255.0;
     return [UIColor colorWithRed:r green:g blue:b alpha:a];
-#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+// [TODO(macOS ISS#2323203)
   } else if ([json isKindOfClass:[NSDictionary class]]) {
     NSDictionary *dictionary = json;
     id value = nil;
     if ((value = [dictionary objectForKey:@"semantic"])) {
       NSString *semanticName = value;
-      NSColor *color = RCTColorFromSemanticColorName(semanticName);
+      UIColor *color = RCTColorFromSemanticColorName(semanticName);
       if (color == nil) {
-        RCTLogConvertError(json, [@"an NSColor.  Expected one of the following values: " stringByAppendingString:RCTSemanticColorNames()]);
+        RCTLogConvertError(json, [@"an UIColor.  Expected one of the following values: " stringByAppendingString:RCTSemanticColorNames()]);
```

Comment:
"a UIColor" but also, this doesn't run on Mac?

Reply:
It is an NSColor on mac, UIColor on iOS via RCTUIKit.h